lua/lspsaga/finder.lua

```diff
@@ -273,7 +273,7 @@ function finder:create_finder_data(result, method)
       start_col = node.col - 10
     end
     node.word = api.nvim_buf_get_text(node.bufnr, node.row, start_col, node.row, node.ecol, {})[1]
-    if node.word:find('^%s') then
```

Comment:
there check the line has indent . so we need check the value of `node.word:find('%S')` is nil or not in sub function